app/Extensions/Backups/Adapter/ResticBackupAdapter.php

```
@@ -0,0 +1,47 @@
+     * @param  array<string, string>  $resticConfig
+     * @param  array<string, string>  $s3Config
+     */
+    public function __construct(
+        private array $resticConfig,
+        private array $s3Config,
+    ) {}
+
+    /*
+     * Provides Wings with the restic info that's been configured in the panel.
+     */
+    public function provideUploadInfo(int $backupSize, Backup $model, Server $server): JsonResponse
```

PHPMD (2.15.0)

Avoid unused parameters such as '$backupSize'. (Unused Code Rules)

(UnusedFormalParameter)